.werft/observability/monitoring-satellite.ts

```diff
@@ -61,49 +62,21 @@ export async function installMonitoringSatellite(params: InstallMonitoringSatell
 }
 
 async function ensureCorrectInstallationOrder(){
-    installSetup()
-
-    deployPrometheus()
-    deployGrafana()
-    deployNodeExporter()
-    deployKubeStateMetrics()
-    deployGitpodServiceMonitors()
-    deployKubernetesServiceMonitors()
-    deployOpenTelemetryCollector()
-}
-
-async function deployPrometheus() {
     const werft = getGlobalWerftInstance()
 
-    werft.log(sliceName, 'installing prometheus')
-    exec('kubectl apply -f observability/monitoring-satellite/manifests/prometheus/', {silent: true})
-    // Prometheus usually takes some time to be created. We sleep to give the operator some time
-    // to create the StatefulSet.
-    exec('sleep 20 && kubectl rollout status statefulset prometheus-k8s', {slice: sliceName})
-}
+    werft.log(sliceName, 'installing monitoring-satellite')
+    exec('hack/deploy-satellite.sh', {slice: sliceName})
```

Comment:
Don't  you need to have observability in the path or `cd` into observability first? :)

Reply:
Do you believe me that I just spend like 20 mins trying to understand why this was failing?

Thanks for the pointer 😄